pkg/server/datastore/fakedatabase.go

```diff
@@ -431,6 +431,28 @@ func (db *FakeDatabase) DeleteRelationship(ctx context.Context, relationshipID u
 	return nil
 }
 
+// WithRelationships overrides all relationships
+func (db *FakeDatabase) WithRelationships(relationships []*entity.Relationship) {
```

Comment:
let's make these variadic functions
```suggestion
func (db *FakeDatabase) WithRelationships(relationships ...*entity.Relationship) {
```